code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -11,11 +11,10 @@
 	siemens_coeff = 0.5 //They thrive on energy
 	payday_modifier = 1.0
 	inherent_traits = list(
-		// TRAIT_NO_UNDERWEAR, // SKYRAT EDIT - LET THEM WEAR PANTIES
+		// TRAIT_NO_UNDERWEAR, // SKYRAT EDIT - LET THEM WEAR PANTIES //BUBBER EDIT - LET THEM HAVE GENDER
```

Comment:
Can you put this commit behind TRAIT_AGENDER?

``//TRAIT_AGENDER - BUBBER EDIT``

Reply:
I am not sure what you mean, I removed TRAIT_AGENDER from that list.

The trait still exists though, its used again for the lustrous variant of ethereals, the pirate variant, which I didn't touch.

Reply:
Do you mean like this?